app/views/projects/index.html.erb

```diff
@@ -16,18 +16,18 @@
     <% end %>
   </div>
 </div>
-<div class='row'>
+<div class='row cta'>
   <div class='col-md-3'>
     <div class="panel panel-default">
       <div class="panel-heading">
         <h2 class="panel-title"><%= fa_icon 'binoculars' %>&nbsp;Discover new software</h2>
       </div>
       <div class="panel-body">
         <p>
-          Search <%= number_with_delimiter(Project.total) %> projects by <%= link_to 'licence'.html_safe, licenses_path %>, <%= link_to 'language'.html_safe, licenses_path %> or <%= link_to 'keyword'.html_safe, keywords_path %>, or explore new, trending or popular projects.
+          Search <%= number_to_human(Project.total) %> projects by <%= link_to 'licence'.html_safe, licenses_path %>, <%= link_to 'language'.html_safe, licenses_path %> or <%= link_to 'keyword'.html_safe, keywords_path %>, or explore new, trending or popular projects. And some text to make this box longer
```

Comment:
PS, I forget, who gets notified of comments? Only the commiter? Or subsrcibers to the thread?